src/common/EvalWorker.res

```diff
@@ -0,0 +1,10 @@
+// Required because workers may receive messages intended for other workers eg. react dev tools
```

Comment:
Ok, stuck again. This module seems to be being executed inside of the window and not actually in a worker. Very confused at this point...

Reply:
All of the examples I've seen online in Webpack5 and Next.js are passing `URL` instances to `new Worker` whereas I am passing `url.toString()`, changing it to be `let make = () => %raw(`new Worker(new URL("./EvalWorker.mjs", import.meta.url))`)` breaks with the following error:

```
webpack/runtime/compat
The "path" argument must be of type string. Received undefined
```

Reply:
hmm, where does `path` come from? weird

Reply:
I found this issue which seems kinda related but is about dynamic imports _inside_ a worker: https://github.com/vercel/next.js/issues/22581

Reply:
That issue is the exact problem and the config workaround does the trick. See my comment here: https://github.com/vercel/next.js/issues/22581#issuecomment-879881706

Definitely not an ideal solution, I'll look into testing the attached PR and hopefully we can get a patch merged.

Reply:
Ok, this is definitely fixed with https://github.com/vercel/next.js/pull/26372

Reply:
For now I've added a monkey patch in the next config...